cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -0,0 +1,96 @@
+# Various Ruby helper methods to generate Varnish VCL from the base configuration.
+
+# Basic regex matcher for the query part of a URL.
+QUERY_REGEX = "(\\?[a-z0-9]+)?$"
```

Comment:
There are additional characters allowed in a query string (e.g. '&' and '%'), does this need to be generalized to handle those?